### DISCUSSION TITLE
Version 0.6.11

### DISCUSSION THREAD

**Discussion by IvyZX** [google/flax#3159] [General]
## What's Changed
* remove unreferenced docs by @chiamp in https://github.com/google/flax/pull/3085
* Bump tensorflow from 2.9.3 to 2.11.1 in /examples/ogbg_molpcba by @dependabot in https://github.com/google/flax/pull/2976
* Bind submodules using __getattribute__ by @cgarciae in https://github.com/google/flax/pull/2901
* Allow manually specifying the rng key for Dropout by @cgarciae in https://github.com/google/flax/pull/3114
* fixed missing separator for rng fold in by @chiamp in https://github.com/google/flax/pull/3095
* Bump imagenet Flax to 0.6.5 by @IvyZX in https://github.com/google/flax/pull/3118
* Rolling back [#3104](https://github.com/google/flax/pull/3104) because internal tests are breaking. by @copybara-service in https://github.com/google/flax/pull/3126
* Use pyproject.toml as hash on CI by @cgarciae in https://github.com/google/flax/pull/3131
* Minor improvements for Sequential by @cgarciae in https://github.com/google/flax/pull/3129
* Update activation.py docstring for PReLU by @levskaya in https://github.com/google/flax/pull/3122
* Eagerly bind submodules by @cgarciae in https://github.com/google/flax/pull/3077
* Add haiku_migration_guide by @cgarciae in https://github.com/google/flax/pull/3139
* Update Haiku migration guide. by @LenaMartens in https://github.com/google/flax/pull/3154
* Fix setup trigger issue under sharing and transforms. by @levskaya in https://github.com/google/flax/pull/3153
* Upgrade the `pjit` guide to `jax.jit` by @IvyZX in https://github.com/google/flax/pull/3016
* Release 0.6.11 by @IvyZX in https://github.com/google/flax/pull/3155

## New Contributors
* @LenaMartens made their first contribution in https://github.com/google/flax/pull/3154

**Full Changelog**: https://github.com/google/flax/compare/v0.6.10...v0.6.11

<hr /><em>This discussion was created from the release <a href='https://github.com/google/flax/releases/tag/v0.6.11'>Version 0.6.11</a>.</em>
